src/operator/quantization/quantized_reshape-inl.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file quantized_reshape-inl.h
+ * \author: Adam Grabowski, adam.grabowski@intel.com
+ */
+
+#ifndef MXNET_OPERATOR_QUANTIZATION_QUANTIZED_RESHAPE_INL_H_
+#define MXNET_OPERATOR_QUANTIZATION_QUANTIZED_RESHAPE_INL_H_
+
+#include <string>
+#include <vector>
+#include "operator/tensor/matrix_op-inl.h"
+#include "operator/numpy/np_matrix_op-inl.h"
+
+namespace mxnet {
+namespace op {
+
+struct QuantizedReshapeParam : public dmlc::Parameter<QuantizedReshapeParam> {
+  mxnet::TShape newshape;
+  mxnet::Tuple<int> shape;
+  bool reverse, keep_highest, is_numpy_op;
+  std::string order;
```

Comment:
I don't think merging both parameter types is a good idea. Duplicating type information in a variable is counter-intuitive. What do you think about separating the operator into two versions: numpy and ndarray one?